### ISSUE TITLE
FB18556966: New NSAccessibility.Role APIs should omit the `Role` suffix (macOS 26 SDK)

### ISSUE THREAD

**Issue by 1024jp** [1024jp/AppleFeedback#47]
The macOS 26 SDK adds some new accessibility roles: `headingRole`[^1],  `listMarkerRole`[^2]  and `webAreaRole`[^3].
These role names have the `Role` suffix, while other existing roles omit it from the Objective-C version APIs. 

[^1]: https://developer.apple.com/documentation/appkit/nsaccessibility-swift.struct/role/headingrole
[^2]: https://developer.apple.com/documentation/appkit/nsaccessibility-swift.struct/role/listmarkerrole
[^3]: https://developer.apple.com/documentation/appkit/nsaccessibility-swift.struct/role/webarearole

Following the Swift naming rule for NSAccessibility.Roles, the trailing `Role` should be omitted.
Therefore, the Swift version APIs for these roles should simply be `heading`, `listMarker`, and `webArea`.

Expected result:
The Swift APIs for the new `NSAccessibilityHeadingRole`, `NSAccessibilityListMarkerRole` and `NSAccessibilityWebAreaRole`, don’t have the `Role` suffix; namely just `heading`, `listMarker`, and `webArea`.

Actual result:
The Swift APIs for the new `NSAccessibilityHeadingRole`, `NSAccessibilityListMarkerRole` and `NSAccessibilityWebAreaRole`, leave the `Role` suffix; namely `headingRole`, `listMarkerRole`, and `webAreaRole`.

Additional note:
This is the same mistake as the NSToolbarItem.Identifier.writingToolsItemIdentifier introduced in the macOS 15.2 SDK (FB15963766).

Environments:
- macOS: Version 26.0 beta 2 (25A5295e)
- Xcode: Version 26.0 beta 2 (17A5241o)

**Comment by 1024jp** [comment]
I found that the same issue also exists in the new NSAccessibility.Subrole API: `.suggestionSubrole`[^1]. The API should be simply `.suggestion`.

[^1]: https://developer.apple.com/documentation/appkit/nsaccessibility-swift.struct/subrole/suggestionsubrole
